benchto-driver/src/main/java/com/teradata/benchto/driver/graphite/GraphiteMetricsLoader.java

```diff
@@ -72,10 +73,10 @@ public void initQueryMetrics()
     }
 
     @Override
-    public List<Measurement> loadMeasurements(Measurable measurable)
+    public CompletableFuture<List<Measurement>> loadMeasurements(Measurable measurable)
```

Comment:
Ok, but as I understand this code, it is serial anyway. Once the caller obtain the `Future` it will be already completed. Am I missing something? So this is why I see no benefit of using it here.